docs/source/migration/3.0.mdx

```diff
@@ -16,7 +16,52 @@ Although most of the library's concepts are the same, many APIs have changed to
 
 This page describes the most important changes, along with how to migrate an existing project from Apollo Android 2.x to Apollo Android 3.x.
 
-## Package name / group id / plugin id
+## The quick route 🚀
+
+Apollo Android 3.0 provides a few shortcuts and compatibility modes to ease the migration from 2.x.  To quickly arrive at a working state, follow these steps:
+
+1. Update your dependencies and imports (`com.apollographql.apollo` → `com.apollographql.apollo3`, see [section below](#package-name--group-id--plugin-id)).  
+  Remove `apollo-coroutines-support` and `apollo-android-support` if applicable.
+2. Gradle configuration:
+```kotlin
+apollo {
+  // Remove this
+  generateKotlinModels.set(true)
+  
+  // Add this
+  useVersion2Compat("com.example")
```

Comment:
```suggestion
  useVersion2Compat()
```

The argument is `rootPackageName`, which most users will not have set  I believe (I know this is confusing, this is why we're making packageName mandatory in 3.x)